subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -395,6 +406,8 @@ int fota_download_cancel(void)
 		send_evt(FOTA_DOWNLOAD_EVT_CANCELLED);
 	}
 
+	download_started = false;
```

Comment:
This will not be set if dowlnoad_client_disconnect() fails, is that intended?

Reply:
If download disconnect fails we still have a running session tbh.

Reply:
but maybe have it above just to be sure.